ksqldb-execution/src/main/java/io/confluent/ksql/execution/plan/PlanInfo.java

```diff
@@ -70,6 +71,10 @@ public PlanInfo merge(final PlanInfo other) {
     return new PlanInfo(allSources.merge(other.allSources));
   }
 
+  public Set<ExecutionStep<?>> getSourceSet() {
```

Comment:
nit:
```suggestion
  public Set<ExecutionStep<?>> getSources() {
```

Reply:
`Sources` is a private class, so I don't think the name is ambiguous. If you prefer we can do `getSourceSteps()` instead, but I do think `getSourceSet()` is no good since the fact that it's a set is not relevant (and can be seen at a glance from the type itself).

Reply:
I think the private class justification is a good one, and `getSources()` is concise.